config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+root 'post_blog#index'
+
+get 'index' => 'post_blog#index'
```

Comment:
Swap the first 'index' here to 'blogs'. The convention for the urls is to always start with the table name.  

get 'blogs' => 'post_blog#index', as: 'blogs'